grammars/jsdoc.cson

```diff
@@ -410,6 +410,10 @@
           }
         ]
       }
+      {
+        'match': '^\\s*\\*(?!/)'
+        'name': 'comment.block.documentation.jsdoc'
```

Comment:
I'm not sure what you mean. Do you want me to add a comment describing this rule? I don't see what this has to do with issue 114.